app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -842,6 +842,7 @@ class AdvancedFilter extends React.Component {
                       value={value.low}
                       type="number"
                       min="0"
+                      max={value.high}
```

Comment:
So unfortunately this has some undesired functionality as is I think. When you have these new bounds, it means you can't click the low value when first creating this filter until you increment the high value (since they're both 0 at the start). It also means you can't decrement the high value without first decrementing the low value, and since this isn't communicated to the user it can feel like it's broken. I'd suggest taking this out for now and just sticking to just the tooltip changes, and then handle this as part of a separate ticket after the PO figures out what the requirements should be from a user perspective.

Reply:
updated